src/Sylius/Bundle/AdminBundle/Resources/views/CatalogPromotion/_form.html.twig

```diff
@@ -29,4 +29,20 @@
     <div class="column">
         {{ translationForm(form.translations) }}
     </div>
+    <script>
```

Comment:
do we need it? it should be configured in Symfony form configuration (probably on `pre_submit` event, if a value is empty form should set proper time)

Reply:
this way the form is empty by default, when user changes date it automatically setups times, so user is aware of what's happening. Leaving it empty by default and then submiting it may lead to misunderstanding of user, where some people may think it should be 23:59 or 00:00, this way we eliminate problem.

Reply:
alright, that was unnecessary, removed that and added seconds to input